### PR TITLE
Conversions: Add expanded triggers.

### PR DESCRIPTION
**Pull request by pjhampton**
## Description of new Instant Answer, or changes

Adds **10,000 new triggering opportunities** (postfixes) such as:

  - cm calculator
  - grams calc
  - fluid ounce converter
  - miles conversion

etc etc

## People to notify

@moollaza 
cc/ @mintsoft @bsstoner 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/conversions

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@pjhampton please resolve the conflict here and I can merge 👍

File: t/Conversions.t [hunk 1]
```diff
@@ -2179,21 +2179,36 @@ ddg_goodie_test(
 			physical_quantity => 'digital'
 		})
 	),
-
-	'centigrade to fahrenheit conversion table' => test_zci(
```
Comment (mintsoft) [inline t/Conversions.t]: Have we lost support for queries `$x to $y conversion table` or is this diff misleading?

Comment (pjhampton) [inline t/Conversions.t]: I removed thinking it was a bad idea. Do you disagree? It's not a regression. I added it a couple of commits back

**Comment by moollaza** [comment]
@pjhampton Thanks! This LGTM :+1:

File: lib/DDG/Goodie/Conversions.pm [hunk 2]
```diff
@@ -151,6 +171,11 @@ handle query => sub {
         };
     }
 
+    my $generic_postfixes = join "|", @generics;
+    my $perserved_query = $_;
```
Comment (moollaza) [inline lib/DDG/Goodie/Conversions.pm]: typo? preserved_query?

**Comment by bsstoner** [comment]
This was released to production 🚀

**Pull request merged**